dev-docs/howto/vpn/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.ipsec.enabled }}
+Required postinstallation steps (also see README.md):
+
+# Configure the LoadBalancer
+
+1. Find the node hosting the VPN server: 
+   kubectl get pods {{ include "..fullname" . }}-frontend-0 -o jsonpath={.spec.nodeName}
+2. Edit the load balancer resource in GCP and remove all other endpoints.
```

Comment:
Can we circumvent this by adding `externalTrafficPolicy: Local` to the service (https://cloud.google.com/kubernetes-engine/docs/concepts/service-load-balancer#health_check). This would also improve failover for when we replace the VPN node (e.g. on Constellation upgrades).

Reply:
fyi I tested this and this seems to work at least on GCP.

Reply:
Thanks, that sounds like the exact solution I needed!